src/syscontainer_build/cli.py

```diff
@@ -88,7 +88,14 @@ def __call__(self, parser, namespace, values, dest, option_string=None):
                         '{} not in a=b format. Skipping...'.format(item))
             subprocess.check_call(ocitools_cmd)
             config_out = os.path.sep.join([output, 'config.json.template'])
-            shutil.move('config.json', config_out)
+            try:
+                with open('config.json', 'r') as config_file:
+                    configuration = json.load(config_file)
+                    configuration['process']['terminal'] = False
+                with open(config_out, 'w') as dest:
+                    json.dump(configuration, dest, indent=8, sort_keys=True)
+            finally:
+                os.unlink('config.json')
```

Comment:
nit: This file will be cleaned up by the ``finally`` block just after this one when the entire temp directory is removed.